src/platform/forms/save-in-progress/FormStartControls.jsx

```diff
@@ -22,7 +22,12 @@ class FormStartControls extends React.Component {
     this.props.router.push(this.props.startPage);
   };
 
+  captureAnalytics = () =>
```

Comment:
Thinking about this some more, I wonder if we even need to log anything in `FormStartControls`? IIRC it's already covered by standard analytics event tracking somewhere higher up in Forms, but I could be wrong. Otherwise, we could also find the right action types and make entries for them in the nifty analytics list you just made for the middleware.

Reply:
Yeah, I think these events are tracked in USFS, but I know they have plans on changing that. Something about opening up hooks for us to use.

It probably makes sense to rely on that logging for now and make the change to track it explicitly when we upgrade USFS to the version where they provide those hooks.

Reply:
The events gathered when starting the form on this branch:
![image](https://user-images.githubusercontent.com/12970166/47312908-7108bc80-d5f2-11e8-92f8-376bd0a71cb9.png)

The `-start` event comes from the addition here. `-sip-form-loaded` comes from our SiP code. I think it's the same for both loading up a saved form and starting a new form (and getting the pre-fill information), so I think we will still need to track form starts after all.

@djmunoz1 to confirm.